dev/tests/bundle/tasks.js

```diff
@@ -0,0 +1,63 @@
+/**
```

Comment:
And a general remark (I haven't checked the code yet) regarding testing gulp – it's just a set of streams. Expose them in functions and test those functions generating streams. Still it takes more code than usual to do that, but it's not that hard.